packages/packages-groonga-org-package-task.rb

```diff
@@ -202,12 +215,9 @@ def download_packages(target_namespace)
           end
         end
       when :windows
-        cd(repositories_dir) do
-          cp(archive, ".")
-          archive_base_name = File.basename(archive)
-          latest_link_base_name = archive_base_name.gsub(@version, "latest")
-          ln_sf(archive_base_name, latest_link_base_name)
-        end
+        latest_link_base_name = target.gsub(@version, "latest")
+        htaccesss_value = "RewriteRule #{latest_link_base_name} #{github_releases_base_url}/v#{@version}/#{target}\n"
+        File.write(htaccess_path, htaccesss_value, mode: "a+")
```

Comment:
Can we avoid reusing `download_packages` for this because the new behavior downloads nothing?

Reply:
I have fixed it.

I have created a new task for createing .htaccess because it seemed that neither `download_packages(target_namespace)` nor `release(target_namespace)` is good method for creating .htaccess in it.
How do you feel that?

**Changes**

* Skip download packages if target_namespace is windows.
* Create new task named `create_htaccess` for each namespaces.
  * I think there is a more suitable name for this task. Do you a good idea for the task name?

Reply:
How about `prepare` or something? It seems that `create_htaccess` is too specific to use in `packages-groonga-org-package-task.rb`.

Reply:
Renamed to `prepare`